resources/services/backup/plans.go

```diff
@@ -0,0 +1,344 @@
+package backup
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/backup"
+	"github.com/aws/aws-sdk-go-v2/service/backup/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/diag"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func Plans() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_backup_plans",
+		Description:  "Contains metadata about a backup plan.",
+		Resolver:     fetchBackupPlans,
+		Multiplex:    client.ServiceAccountRegionMultiplexer("backup"),
+		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter: client.DeleteAccountRegionFilter,
+		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"arn"}},
```

Comment:
I think we should prefer id over arn (if it's globally unique)

Reply:
I would leave it as ARN. When they say global it could mean global to that account or that partition, not global to all of AWS